app/timetables/factories.py

```diff
@@ -0,0 +1,161 @@
+import datetime
+
+import factory
+from django.contrib.auth import get_user_model
+from django.utils.text import slugify
+from factory.django import DjangoModelFactory
+
+from . import models
+from common.mixins import TimestampMixin
+
+
+class TimestampFactory(DjangoModelFactory):
+    """TimestampMixin model factory."""
+
+    class Meta:
+        model = TimestampMixin
+        abstract = True
+
+    date_created = datetime.datetime(2008, 1, 1, 0, 0, 0, tzinfo=datetime.timezone.utc)
+    date_modified = datetime.datetime(2009, 1, 1, 0, 0, 0, tzinfo=datetime.timezone.utc)
+
+
+class WeekdayFactory(DjangoModelFactory):
+    """Weekday model factory."""
+
+    class Meta:
+        model = models.Weekday
+
+    name = 'Monday'
+    slug = factory.LazyAttribute(lambda obj: '%s' % slugify(obj.name))
+
```

Comment:
I can abstract the slug field just like the SlugifyMixin class, but abstracting name alongside is not going to help. The factories should be built to fit how our models were written. Having the same name written everywhere will not provide clarity when running the test. That said, I could try putting the slug field in an abstract factory.